src/labels_transfer/xgboost/config.vsh.yaml

```diff
@@ -31,6 +31,11 @@ functionality:
           description: Output directory for model
           direction: output
           required: false
+        - name: "--output_uns_parameters"
+          type: string
+          default: "xgboost_parameters"
+          description: The key in `uns` slot of the output AnnData object to store the parameters of the XGBoost classifier.
+          required: false
```

Comment:
Do any of these output slots determine the behavior of downstream components (like with log transformation in scanpy?)

Reply:
They don't, but I thought it would be nice to keep track of what happened. Might be convenient for hyperparameter tuning, for example

Reply:
Ok, no worries adding this then :)